src/content/packages-and-plugins/swift-package-manager/for-app-developers.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Swift Package Manager for app developers
+description: How to use Swift Package Manager for native iOS or macOS dependencies
+---
+
+:::warning
+Flutter is migrating to [Swift Package Manager][] to manage iOS and macOS native
+dependencies.
+This is an experimental feature that might change in the future.
+It is currently only available on the [`main` channel][].
+Flutter will continue to support CocoaPods until further notice.
+:::
+
+Flutter's Swift Package Manager integration has several benefits:
+
+1. **Access to the Swift package ecosystem**.
+   Flutter plugins can use the growing ecosystem of [Swift packages][]! 
+2. **Simplifies Flutter installation**.
+   Swift Package Manager is bundled with Xcode.
+   In the future, you won’t need to install Ruby and CocoaPods to target iOS or
+   macOS.
+
+If you find a bug in Flutter's Swift Package Manager support,
+please [open an issue][].
+
+[Swift Package Manager]: https://www.swift.org/documentation/package-manager/
+[`main` channel]: /release/upgrade#switching-flutter-channels
+[Swift packages]: https://swiftpackageindex.com/
+[open an issue]: {{site.github}}/flutter/flutter/issues/new?template=2_bug.yml
+
+{% include docs/swift-package-manager/how-to-enable-disable.md %}
+
+## How to add Flutter to an existing app (add-to-app)
+
+Flutter's Swift Package Manager support doesn't work with add-to-app scenarios
+yet.
+
+For status updates, see [flutter#146957].
```

Comment:
**issue:** Keep the documentation timeless per [GSG](https://developers.google.com/style/timeless-documentation).

```suggestion
Flutter's Swift Package Manager support doesn't work with add-to-app scenarios.

To keep current on status updates, consult [flutter#146957].
```

Reply:
Updated!